kaolin/graphics/SoftRenderer.py

```diff
@@ -664,7 +640,37 @@ def textures_to_faces(self, textures, faces):
         """
         B = textures.shape[0]
         F = textures.shape[1]
-        device = vertices.device
+        device = textures.device
         faces = faces + (torch.arange(B, device=device) * F)[:, None, None]
         textures = textures.reshape(B * F, 2, 3)
         return textures[faces]
+
+    def set_eye_from_angles(self, distance: Union[int, float, torch.Tensor],
+                            elevation: Union[int, float, torch.Tensor],
+                            azimuth: Union[int, float, torch.Tensor],
+                            degrees: bool = True):
+        r"""Compute the position of the camera given azimuth and elevation angles,
+        and the distance of the camera from the origin (i.e., spherical coordinates).
+
+        Args:
+            distance (int, float, or torch.Tensor): Distance of the camera from the origin.
+            azimuth (int, float, or torch.Tensor): Azimuth angle.
+            elevation (int, float, or torch.Tensor): Elevation angle.
```

Comment:
I don't think supporting `int` makes sense

Reply:
I think it does. Eg. people can often mistakenly pass `distance=2` when they mean 2 meters, or `azimuth=90` when they mean 90 degrees

Reply:
Sorry, what I mean is explicitly saying that we will support int, user should understand that the input will be interpreted as a float (if it can).

For instance https://pytorch.org/docs/stable/torch.html#torch.pow only says `float or tensor` but in fact `int` works just fine. It make the doc lighter.

Reply:
Makes sense